vnext/overrides.json

```diff
@@ -415,7 +415,7 @@
       "type": "derived",
       "file": "src/Libraries/Text/Text.windows.js",
       "baseFile": "Libraries/Text/Text.js",
-      "baseVersion": "0.0.0-30a89f30c",
+      "baseVersion": "0.0.0-a36d9cd7e",
```

Comment:
Was there a merge conflict here?

Reply:
I merged the text border PR this morning and may have used a wrong version there - updated it to match all others in this PR

Reply:
It does look like there was also a change that happened to Text that we'll want to pick up. See https://github.com/facebook/react-native/commit/0fda91ffffa4972ebe58e3d0b610692a1286eaa1

If you run `yarn validate-overrides` I think the tool will likely complain because the base hash has changed since you added Text.windows.js in your last PR. CI should also catch this. If you leave the previous baseVersion, you should be able to run `npx react-native-platform-override upgrade` in `vnext`, to automatically update the manifest and pull in any changes.